modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/ReturnDelegate.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.runtime;
+
+import org.mule.extension.runtime.OperationContext;
+import org.mule.extension.runtime.OperationExecutor;
+
+/**
+ * A delegate interface to decouple a {@link OperationExecutor}'s return value
```

Comment:
you should get it by continuing reading, but don't bother because per prior discussions all of this will change